ganttproject/src/net/sourceforge/ganttproject/export/ExporterToExcel.java

```diff
@@ -0,0 +1,54 @@
+package net.sourceforge.ganttproject.export;
+
+import biz.ganttproject.core.option.GPOptionGroup;
+import net.sourceforge.ganttproject.language.GanttLanguage;
+
+import java.awt.*;
+import java.io.File;
+import java.util.List;
+
+/**
+ * Created by valentin on 15.03.17.
+ */
+public class ExporterToExcel extends ExporterBase {
+
+  private static String[] FILE_EXTENTION = new String[] { ".xlsx" };
+
+  @Override
+  public String getFileTypeDescription() {
+    return GanttLanguage.getInstance().getText("");//add description in prorerty files and paste key here
```

Comment:
If you meant to add todo to yourself, please prefix the comment with `TODO:`
Also, add a space between `;` and `//`.
Typo: s/prorerty/property